apps/products-feed/src/modules/file-storage/s3/urls-and-names.ts

```diff
@@ -0,0 +1,28 @@
+import { S3BucketConfiguration } from "../../app-configuration/app-config";
+
+interface GetDownloadUrlArgs {
+  s3BucketConfiguration: S3BucketConfiguration;
+  saleorApiUrl: string;
+  channel: string;
```

Comment:
can we call it `channelSlug` or `channelId`? `string` is not expressive enough

Reply:
I'll address that in a separate PR. Issue created: https://github.com/saleor/apps/issues/496

Need to resolve critical issues ASAP, it will be faster to change after the fixes are merged.